curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-advanced-array-methods-by-building-a-statistics-calculator/65ca2d0625aa3a3201067f70.md

```diff
@@ -0,0 +1,148 @@
+---
+id: 65ca2d0625aa3a3201067f70
+title: Step 30
+challengeType: 0
+dashedName: step-30
+---
+
+# --description--
+
+In `.forEach()`'s callback, use the `el` parameter to access the `counts` object. If the element is not in `counts`, set it to 0. Otherwise, increment the count for the element. Log out the final result of `counts`.
```

Comment:
a few grammar fixes and missing backticks

```suggestion
In the callback function of your `forEach`, use the `el` parameter to access the `counts` object. If the element is not in `counts`, set it to `0`. Otherwise, increment the count for the element. 

Below your `forEach`, log out the final result of `counts`.
```